modelscope/pipelines/cv/image_depth_estimation_marigold_pipeline.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+from typing import Any, Dict, Union
+
+from PIL import Image
+
+from modelscope.metainfo import Pipelines
+from modelscope.outputs import OutputKeys
+from modelscope.pipelines.base import Input, Model, Pipeline
+from modelscope.pipelines.builder import PIPELINES
+from modelscope.utils.constant import Tasks
+from modelscope.utils.logger import get_logger
+
+logger = get_logger()
+
+
+@PIPELINES.register_module(
+    Tasks.image_depth_estimation, module_name=Pipelines.image_depth_estimation_marigold)
+class ImageDepthEstimationMarigoldPipeline(Pipeline):
+
+    def __init__(self, model: str, **kwargs):
```

Comment:
pls refer to https://github.com/modelscope/modelscope/blob/ade394d68c808b257028a961fc1bfa940c6bf7db/modelscope/pipelines/cv/card_detection_pipeline.py#L20 to add docstr

Reply:
added